hws/src/dr_ste.py

```diff
@@ -295,6 +296,21 @@ def dump_fields(self, verbosity, tabs):
 
         return _str
 
+    def dump_miss(self, verbosity, tabs, expected_miss_index):
+        if not len(self.fields_dic):
+            return ''
+
+        miss_is_expected_loc = (
+            self.miss_loc.gvmi_str == _config_args.get("vhca_id")
+            and
+            hex(self.miss_loc.index) == expected_miss_index
+        )
+        if miss_is_expected_loc and verbosity < 4:
+             return ''
+
+        expected_str = '' if miss_is_expected_loc else ' (NOT expected value)'
```

Comment:
It turns out that this condition is triggered when there are hash collisions, and the first entry in the bucket misses to the 2nd entry for example. @hamdanigbaria I'm not sure how to enhance this message so it doesn't trigger for the "execute along the miss chain" case.